### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by o---** [DirectoryLister/DirectoryLister#546] [Help & Support] [answered]
How to add another translation? I have tried by copying en.yaml into cs.yaml but I am getting _Unknown value 'cs' for configuration option 'language'_ laravel error.

**Comment by PHLAK** [comment, marked as answer]
Try clearing the cache (`rm -r app/cache/*`) after making your changes.

**Reply by o---** [reply]
Thanks, that worked!

**Reply by o---** [reply]
sk.yaml
```home: Domov
download: Stiahnuť tento priečinok
search: Vyhľadávanie
file:
  name: Názov súboru
  size: Veľkosť
  date: Dátum
  info: Informácie o súbore
powered_by: Powered by
scroll_to_top: Scroll to Top
toggle_theme: Prepnúť Svetlý/Tmavý režim

error:
  directory_not_found: Priečinok neexistuje
  file_not_found: Súbor sa nenašiel
  file_size_exceeded: Súbor je príliš veľký
  no_results_found: Nič sa nenašlo
  unexpected: Nastala neočakávaná chyba

enable_debugging: Zapni ladenie pre získanie dodatočných informácií

**Reply by PHLAK** [reply]
Thanks! Would you be able to create a pull request for this? I'd love to add this in.

**Reply by o---** [reply]
Sorry for being late. I just saw your message, but I see you have already added the translation.
